### PR TITLE
german localization rev. 4

### PR DESCRIPTION
**Pull request by Gordon-Dry**

### CODE REVIEW

File: strings.lua [hunk 1]
```diff
@@ -261,8 +261,8 @@ Lifts, stairs and underground entrances are in the assets, search for "freestyle
 			["WallConcreteModernDesc"] = "Concrete wall, also suitable for tunnels. Add/remove supporting wall with <m> and <n>.",
 			["WallConcretePlainName"] = "Concrete wall, plain",
 			["WallConcretePlainDesc"] = "Concrete wall, also suitable for tunnels. Add/remove supporting wall with <m> and <n>.",
-			["WallFenceMattoniName"] = "Brick fence",
-			["WallFenceMattoniDesc"] = "Brick fence, also suitable for tunnels. Add/remove supporting wall with <m> and <n>.",
+			["WallFenceMattoniName"] = "Brick wall",
```
Comment (lolluslollus) [inline strings.lua:264]: Some of these things are walls, others are fences, so don't swap them.

File: strings.lua [hunk 2]
```diff
@@ -261,8 +261,8 @@ Lifts, stairs and underground entrances are in the assets, search for "freestyle
...
+			["WallFenceMattoniDesc"] = "Brick wall, also suitable for tunnels. Add/remove supporting wall with <m> and <n>.",
```
Comment (lolluslollus) [inline strings.lua:265]: Some of these things are walls, others are fences, so don't swap them.

File: strings.lua [hunk 3]
```diff
@@ -729,16 +729,16 @@ Aufzüge, Treppen und unterirdische Eingänge finden Sie in den Assets, suchen S
 			["WallConcreteModernDesc"] = "Betonwand, auch für Tunnels geeignet. Stützwand mit <m> und <n> einstellen.",
 			["WallConcretePlainName"] = "Betonwand, schlicht",
 			["WallConcretePlainDesc"] = "Betonwand, auch für Tunnels geeignet. Stützwand mit <m> und <n> einstellen.",
-			["WallFenceMattoniName"] = "Backsteinzaun",
-			["WallFenceMattoniDesc"] = "Backsteinzaun, auch für Tunnels geeignet. Stützwand mit <m> und <n> einstellen.",
+			["WallFenceMattoniName"] = "Backsteinmauer",
```
Comment (lolluslollus) [inline strings.lua:732]: Some of these things are walls, others are fences, so don't swap them.

File: strings.lua [hunk 4]
```diff
@@ -729,16 +729,16 @@ Aufzüge, Treppen und unterirdische Eingänge finden Sie in den Assets, suchen S
...
+			["WallFenceMattoniDesc"] = "Backsteinmauer, auch für Tunnels geeignet. Stützwand mit <m> und <n> einstellen.",
```
Comment (lolluslollus) [inline strings.lua:733]: Some of these things are walls, others are fences, so don't swap them.

File: strings.lua [hunk 5]
```diff
@@ -729,16 +729,16 @@ Aufzüge, Treppen und unterirdische Eingänge finden Sie in den Assets, suchen S
...
 			["WallIronName"] = "Eisenzaun",
 			["WallIronDesc"] = "Eisenzaun, auch für Tunnel geeignet. Stützwand mit <m> und <n> einstellen.",
 			["WallIronGlassCopperName"] = "Wand aus Eisen und Glas",
 			["WallIronGlassCopperDesc"] = "Eisen- und Glaswand, auch für Tunnels geeignet. Stützwand mit <m> und <n> einstellen.",
-			["WallStaccionataFsName"] = "1,4m Betonzaun",
-			["WallStaccionataFsDesc"] = "1,4m Betonzaun, auch für Tunnel geeignet. Stützwand mit <m> und <n> einstellen.",
-			["WallStaccionataFsTallName"] = "2m Betonzaun",
-			["WallStaccionataFsTallDesc"] = "2m Betonzaun, auch für Tunnel geeignet. Stützwand mit <m> und <n> einstellen.",
+			["WallStaccionataFsName"] = "1,4m Betonwand",
```
Comment (lolluslollus) [inline strings.lua:738]: Some of these things are walls, others are fences, so don't swap them.

File: strings.lua [hunk 6]
```diff
@@ -729,16 +729,16 @@ Aufzüge, Treppen und unterirdische Eingänge finden Sie in den Assets, suchen S
...
+			["WallStaccionataFsDesc"] = "1,4m Betonwand, auch für Tunnel geeignet. Stützwand mit <m> und <n> einstellen.",
```
Comment (lolluslollus) [inline strings.lua:739]: Some of these things are walls, others are fences, so don't swap them.

File: strings.lua [hunk 7]
```diff
@@ -729,16 +729,16 @@ Aufzüge, Treppen und unterirdische Eingänge finden Sie in den Assets, suchen S
...
+			["WallStaccionataFsTallName"] = "2m Betonwand",
```
Comment (lolluslollus) [inline strings.lua:740]: Some of these things are walls, others are fences, so don't swap them.

File: strings.lua [hunk 8]
```diff
@@ -729,16 +729,16 @@ Aufzüge, Treppen und unterirdische Eingänge finden Sie in den Assets, suchen S
...
+			["WallStaccionataFsTallDesc"] = "2m Betonwand, auch für Tunnel geeignet. Stützwand mit <m> und <n> einstellen.",
```
Comment (lolluslollus) [inline strings.lua:741]: Some of these things are walls, others are fences, so don't swap them.

File: strings.lua [hunk 9]
```diff
@@ -864,7 +864,7 @@ Aufzüge, Treppen und unterirdische Eingänge finden Sie in den Assets, suchen S
 			["WaypointsTooCloseToStation"] = "Zu nahe an einem bestehenden Bahnhof oder Depot. Schneiden Sie das Gleis mit dem Gleisteiler, wenn Sie näher herankommen müssen. Die Bahnsteigköpfe verbinden sich automatisch, wenn sie näher als 7 Meter sind",
 			["WaypointsTooFar"] = "Zu weit von seinem Zwilling entfernt",
 			["WaypointsWrong"] = "Platziere zwei Marker auf demselben Bahnsteig und zwei auf demselben Gleis",
-			["WrongTrack"] = "Bahnsteig und / oder Gleis sind nicht sauber, verlegen Sie sie neu.",
+			["WrongTrack"] = "Bahnsteig und / oder Gleis sind zu ungenau, verlegen Sie sie neu.",
```
Comment (lolluslollus) [inline strings.lua:867]: Nicht sauber ist nicht sauber, es gibt keine genaue oder ungenaue Gleise.

**Review by lolluslollus — CHANGES_REQUESTED** [review]
Some of these things are walls, others are fences, so don't swap them.

File: strings.lua (hunk 7 above)
Comment (Gordon-Dry) [reply strings.lua:740]: Allright.
The thing is, the german word "Zaun" is not meant to be used for something made of bricks or concrete.
If it's at the the side of a tunnel, it's a wall ("Wand"), if it's in the middle as a divider, it's a wall ("Mauer" or even better: "Trennwand" which literally means "divider wall").

File: strings.lua (hunk 9 above)
Comment (Gordon-Dry) [reply strings.lua:867]: "Unsauber verlegt" oder "nicht sauber verlegt" ist korrektes Deutsch. Das hat die selbe Bedeutung wie "nicht genau" oder "ungenau".
Deutsch ist meine Muttersprache.

**Pull request merged**